distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -400,6 +400,11 @@ grant codeBase "file:/security-core-api/security-sts-attributequeryclaimshandler
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}-", "read";
 }
 
+grant codeBase "file:/sync-installer-impl" {
```

Comment:
After further analysis, it would seem that the sync-installer is now a proxy in between our code and `org.apache.karaf.features.core` as such, it should be given all the same permissions. Now the permissions that are there might not be sufficient for all possible features installed manually because at the end of the day, these 2 bundles are on the stack (or combined) when the feature is being installed which means we should probably be giving all the permissions to these 2 bundles but that is a different issue.

For now, I would suggest that you add the sync-installer domain to the ```
grant codeBase "file:/org.apache.karaf.features.core" {
    permission java.io.FilePermission "${ddf.home.perm}etc${/}-", "read,write";
    permission java.io.FilePermission "${ddf.home.perm}examples", "read,write";
    permission java.io.FilePermission "${ddf.home.perm}examples${/}-", "read,write";
}```